src/nvme/private.h

```diff
@@ -82,6 +82,7 @@ struct nvme_ctrl {
 	char *subsysnqn;
 	char *traddr;
 	char *trsvcid;
+	char *dhchap_key;
```

Comment:
`dhchap_host_key`?

Reply:
We could, but with this naming we're consistent with the kernel usage. But in the end, I don't mind.